React/CoreModules/RCTDevSettings.mm

```diff
@@ -181,7 +187,7 @@ - (void)initialize
     _bridgeExecutorOverrideToken = [[RCTPackagerConnection sharedPackagerConnection]
         addNotificationHandler:^(id params) {
           if (params != (id)kCFNull && [params[@"debug"] boolValue]) {
-            weakBridge.executorClass = objc_lookUpClass("RCTWebSocketExecutor");
```

Comment:
Why did this change? IIRC this was used to switch the executor to the (legacy) chrome debugger.

Reply:
[Here](https://github.com/facebook/react-native/blob/main/React/CoreModules/RCTDevSettings.mm#L343) we're setting `self.executorClass` to `objc_getClass("RCTWebSocketExecutor")`, I though it would be nice refactor.

Reply:
That's only the case if `self.isRemoteDebuggingAvailable && self.isDebuggingRemotely`. That methods serves the same purpose as this handler, to enable remote debugging.

If you wanted to refactor (but that probably should be a different PR), you could try reusing `setIsDebuggingRemotely` here.